Update limitations.asciidoc (#765)
vaadin/docs: articles/tools/mpr/configuration/limitations.asciidoc

Therefore, for example, if you have a modal Vaadin 14 Dialog, which has a Vaadin 8 ComboBox inside, the popup of the Vaadin 8 ComboBox will open under the Vaadin 14 Dialog's modal curtain.
Other similar glitches may occur and solving them requires code refactoring.

== Views with a lot of component might be slower to render 

> MPR applications have usually roughly the double amount of styles compared to applications that use only one Vaadin version.

Reviewer: ```suggestion
MPR applications usually have roughly the double amount of styles compared to applications that use only one Vaadin version.
```

Therefore, for example, if you have a modal Vaadin 14 Dialog, which has a Vaadin 8 ComboBox inside, the popup of the Vaadin 8 ComboBox will open under the Vaadin 14 Dialog's modal curtain.
Other similar glitches may occur and solving them requires code refactoring.

== Views with a lot of component might be slower to render 

> MPR applications have usually roughly the double amount of styles compared to applications using only one Vaadin version. This will make the computed style calculation (part of the rendering) of the view slower since every element has to be checked against a bigger number of selectors. This is likely to affect especially views where there are a huge number of components visible at the same time. It is possible to optimize this by reducing the total number of selectors, reducing their complexity, or reducing the number of components visible.

Reviewer: ```suggestion
MPR applications have usually roughly the double amount of styles compared to applications that use only one Vaadin version.
This will slow down the computed style calculation of the view, which is part of rendering the view, since every element has to be checked against a bigger number of selectors.
This is likely to affect especially views that have a huge number of components visible at the same time.
It is possible to optimize this by reducing the total number of selectors, their complexity, or the number of visible components.
```
